tests/tests.sh.in

```diff
@@ -260,7 +260,8 @@ test_lookup fails rdata ip ThisIsNotAnIP
 #  17  "\x02\xc0\xa8\x01\x01\x01\x03dom\x07next...  192.168.1.1                A       a.next_ip.dom.
 #  18  "\x02\xc0\xa8\x01\x01\x01\x03dom\x00\x00...  c0a8:101:103:646f:6d20::   AAAA    aaaa-6.next_ip.dom.
 #  19  "\x02\xc0\xa8\x01\x01\x01\x03dom\x80\x00...  c0a8:101:103:646f:6d80::   AAAA    aaaa-7.next_ip.dom.
-#  20  "\x02\xc0\xa8\x01\x01\x01\x07next_ip\x01a... 192.168.1.1                A       a.next_ip.
+#  20  "\x02\xc0\xa8\x01\x01\x01\x07next_ip\x00..   192.168.1.1                A       next_ip.
+#  21  "\x02\xc0\xa8\x01\x01\x01\x07next_ip\x01a... 192.168.1.1                A       a.next_ip.
```

Comment:
subsequent entries need to be renumbered

Reply:
Fixed. Will squash when merging.